src/simulation_strategies/FixedLossSimulationStrategy.py

```diff
@@ -46,35 +47,24 @@ def __randomize_modes_phases(state_in_modes_basis: ndarray) -> ndarray:
             :param state_in_modes_basis: A given state in modes basis.
             :return: Given mode state with randomized phases.
         """
-        randomized_phases_state = []
-
-        for mode in state_in_modes_basis:
-            phi = random() * 2 * pi
-            randomized_phases_state.append(exp(1j * phi) * mode)
-
+        randomized_phases_state = exp(1j * rand(len(state_in_modes_basis))) * state_in_modes_basis
         return randomized_phases_state
 
     @staticmethod
     def __calculate_probabilities(state: ndarray) -> ndarray:
-        probabilities = []
-        for detector in state:
-            probabilities.append(conjugate(detector) * detector)
+        probabilities = [conjugate(detector) * detector for detector in state]
         return probabilities
 
     def __calculate_approximation_of_boson_sampling_outcome(self, probabilities: ndarray) -> ndarray:
         """
             This method applies evolution to every photon. Note, that evolution of each particle is independent of
             each other.
             :param probabilities:
-            :return:
+            :return: An output array that contains the number of observations for each photon left (histogram)
```

Comment:
**Needs reviewing**: From my point of view an histogram is returned.

Reply:
That's incorrect. What is returned here is a boson sampling state described by first quantization (that is mode assignment basis). E.g. a state [3, 6, 1] would mean we've got a 3-particle state, with the first particle being in the third mode, the second being in the sixth, and the last one being in the first mode. That can by no means be interpreted as a histogram. You can find more information in referenced works. Thanks for the vigilance ;)

Reply:
I've added the description to the return section.